### PR TITLE
84 add travis ci

### LINKED ISSUE #84: Add Travis CI

### PR DESCRIPTION
**Pull request by threez**
closes #84

### CODE REVIEW

**Comment by threez** [comment]
@Teelevision

**Comment by threez** [comment]
please review @Teelevision

File: Makefile [hunk 1]
```diff
@@ -11,6 +11,15 @@ export JAEGER_SAMPLER_TYPE:=const
 export JAEGER_SAMPLER_PARAM:=1
 export LOG_FORMAT:=console
 
+test:
+	go test -mod vendor -count=1 -v -cover -race -short ./...
+
+integration:
+	go test -mod vendor -count=1 -v -cover -race -run TestIntegration ./...
+
+ci:
```
Comment (Teelevision) [inline Makefile]: Add `ci` to `.PHONY` and move `install` back to the top so that it stays the default make task.

Comment (threez) [reply Makefile]: done

**Pull request merged**